docs/content/en/docs/concepts/architecture/components/metrics-operator/_index.md

```diff
@@ -7,17 +7,31 @@ cascade:
 ---
 
 
-The Keptn Metrics Operator collects, processes, and analyzes metrics data from a variety of sources.
-Once collected, this data, can be used to generate a variety of reports and dashboards
-that provide insights into the health and performance of the application and infrastructure.
-
-While Kubernetes does have two metrics servers, they have limitations.
-The custom and external APIs only allow you to use a single observability platform.
-The Keptn Metrics Operator solves this problem by providing a single entry point for
-all your metrics data, regardless of its source.
-Furthermore, due to the integration with the Kubernetes custom metrics API, these metrics are also
-compatible with the Kubernetes HorizontalPodAutoscaler (HPA) which enables the horizontal scaling of workloads
-based on metrics collected from multiple observability platforms such as Prometheus, Dynatrace or Datadog.
+The Keptn Metrics Operator collects, processes,
+and analyzes metrics data from a variety of sources.
+Once collected, this data can be used
+to generate a variety of reports and dashboards
+that provide insights into the health and performance
+of the application and infrastructure.
+
+While Kubernetes does have two ways to extend the metric API server, they have limitations,
+especially that their custom and external APIs
+only allow you to use a single instance of a single observability platform
```

Comment:
I would not go into so much detail here, especially cus the info is not 100% correct.
```suggestion
While Kubernetes does have ways to extend its metrics APIs, they have limitations,
especially that they
only allow you to use a single observability platform
```